guides/SPI.md

```diff
@@ -21,6 +21,21 @@ In order to become a _service provider_, you have two options:
 * Register an application in the Wix Developer Center, and add the SPI component that you wish to implement. 
 * Become a _service provider_ in a single site by implementing the SPI using Velo. 
 
+## Integration with Wix
+In order to enable Wix applications to communicate with your provider application, Wix needs to know some configuration details. You add the configurations via the Dev Center by entering your app’s url and method and other configuration details in the **JSON Editor** in the service’s extension description page. 
```

Comment:
> In order to enable Wix applications to communicate with your ~~provider~~ application

_provider application_ isn't a term I've seen yet. _application_ or _service_ are both fine in this case.

> by entering ~~your app’s url and method and other~~ ^the requested^ configuration details 

Not sure what "url and method" means, but the sentence works just as well without it.

> in the **JSON Editor**

In this case, we're not describing an interaction on a page, so it's better not to bold

>  in the service’s extension description page. 

What does this mean? I don't see it represented anywhere on the screen.